app/src/modules/win.js

```diff
@@ -2,7 +2,7 @@ import localeHandler from './localeHandler';
 import menu from '../menu';
```

Comment:
NRTTL:
Adding this 
```
        <a target="_blank" href="mailto:mobiledev@lisk.com" rel="noopener noreferrer">
          mailto:mobiledev@lisk.com
        </a>
 ```
 It still opens a new window when clicked, is that supposed to happen?

Reply:
yhea reason is because the navigation events on electron isn't triggered when the <a /> is action on